account_invoice_consolidated/migrations/14.0.1.0.1/post-migrate.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tools.sql import column_exists
+
+
+def migrate(cr, version):
+    if not column_exists(cr, "account_move", "old_invoice_id"):
```

Comment:
FMI, when does this old_invoice_id supposed to be filled ?

Reply:
said in line 9. during account migration, in OU

Reply:
(facepalm) ;-) thanks
I'll learn to better read